instrumentation/opentelemetry-instrumentation-logging/README.rst

```diff
@@ -13,6 +13,140 @@ Installation
 
     pip install opentelemetry-instrumentation-logging
 
+Log record enrichment behavior of logging instrumentation
+
+"""
+The OpenTelemetry ``logging`` integration automatically injects tracing context into log statements.
+It's structured within an `if` statement that checks  [`set_logging_format`] is `True`, indicating that logging configuration should be applied.
+
+The integration registers a custom log record factory with the the standard library logging module that automatically inject
+tracing context into log record objects. Optionally, the integration can also call ``logging.basicConfig()`` to set a logging
+format with placeholders for span ID, trace ID and service name.
+
+1. **Determining the Log Format**: The code first attempts to determine the logging format by looking for a `logging_format` key in the [`kwargs`]( dictionary. If it's not found, it tries to fetch the value from an environment variable named [`OTEL_PYTHON_LOG_FORMAT`]. If neither is available, it defaults to a predefined constant [`DEFAULT_LOGGING_FORMAT`]. This approach provides flexibility, allowing the log format to be specified through function arguments, environment variables, or falling back to a default if neither is provided.
```

Comment:
No need for this level of detail for implementation. I believe it is enough to have the related env vars included in the README (as you have below).